assets/css/style.css

```diff
@@ -105,4 +183,30 @@
     width: 100px;
     margin: 10px;
     background-color: red;
+}
+
+/* Responsive */
+/* Untuk Tablet */
+@media all and (max-width: 1000px) {
+  
+}
+
+/* Untuk Smartphone */
+@media all and (max-width: 670px) {
```

Comment:
Please use `@media only screen` to avoid **undetected** problems from unsupported device, that is a printer.

Reply:
Please do not use media queries for smartphone.

Again, to avoid further confusion from other contributors which do not use GitHub at the moment. Please avoid to implemented value above the `/* TO DO */` comment.